### DISCUSSION TITLE
Meal plan period configurable

### DISCUSSION THREAD

**Discussion by terrapinlogo** [mealie-recipes/mealie#1225] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Allow the user to configure the period/slice used for the meal plan. eg weekly, biweekly, monthly.

Users may go shopping/plan based on pay day
users might live out in country and only want to drive in biweekly or monthly

**(Optional): Suggest A Solution**
_Replace This Text: A concise description of your preferred solution. Things to address include:_
* Details of the technical implementation
* Tradeoffs made in design decisions
* Caveats and considerations for the future

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
This is available now: https://github.com/mealie-recipes/mealie/pull/3650
